grammar.js

```diff
@@ -1415,18 +1421,18 @@ module.exports = grammar({
         '--',
         '^',
         '~'
-      ].map(operator => seq(operator, $._expression)))),
+      ].map(operator => seq(operator, field('expression', $._expression))))),
 
-    _pointer_indirection_expression: $ => prec.right(PREC.UNARY, seq('*', $._expression)),
+    _pointer_indirection_expression: $ => prec.right(PREC.UNARY, seq('*', field('expression', $._expression))),
```

Comment:
This production (because it starts with `_`) doesn't introduce an intermediate node, so I don't think it should introduce a field at all. That is, assuming a field is already created at the use site...